lib/autoInject.js

```diff
@@ -4,10 +4,10 @@ import arrayMap from 'lodash/_arrayMap';
 import clone from 'lodash/_copyArray';
 import isArray from 'lodash/isArray';
 
-var argsRegex =  /^function\s*[^\(]*\(\s*([^\)]*)\)/m;
+var argsRegex = /^(function[^\(]*)?\(?\s*([^\)=]*)/m;
 
 function parseParams(func) {
-    return  func.toString().match(argsRegex)[1].split(/\s*\,\s*/);
+    return func.toString().match(argsRegex)[2].trim().split(/\s*\,\s*/);
```

Comment:
This is not about you or your reputation.  You have a valid opinion and I encourage you to raise it for discussion.  Just not in the patch comments of this tangentially-related PR.